modules/app.xqm

```diff
@@ -6,9 +6,10 @@ xquery version "3.1";
 
 module namespace app="http://exist-db.org/xquery/app";
 
+
+import module namespace templates="http://exist-db.org/xquery/templates";
```

Comment:
They had been alphabetical by namespace prefix. If you want to group external module imports together, let's add a space between the groups.

Reply:
I've applied this idea in a branch. Marking this as resolved in advance.